From b363a3990d880778666a06b7816c76771e400c6e Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Thu, 14 Mar 2013 11:21:29 +0200
Subject: [PATCH 26/63] stob: don't use M0_UT_ASSERT() ad-ub and adieu-ub

UT asserts cannot be used in UB code. They result in run-time errors
similar to this one:

$ sudo utils/ub.sh -t ad-ub
round  1 ad-ub[lt-ub: TestRun.c:168: CU_assertImplementation: Assertion `((void *)0) != f_pCurSuite' failed.
.utils/ub.sh: line 40:  9924 Aborted                 /mnt/hgfs/vvv/src/mero/utils/$EXEC "$@"
---
 stob/ut/ad.c    | 2 +-
 stob/ut/adieu.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/stob/ut/ad.c b/stob/ut/ad.c
index c11bac4..7a9f316 100644
--- a/stob/ut/ad.c
+++ b/stob/ut/ad.c
@@ -249,7 +249,7 @@ static void test_write(int i)
 	struct m0_fol_rec_part *fol_rec_part;
 
 	M0_ALLOC_PTR(fol_rec_part);
-	M0_UT_ASSERT(fol_rec_part != NULL);
+	M0_UB_ASSERT(fol_rec_part != NULL);
 
 	m0_stob_io_init(&io);
 
diff --git a/stob/ut/adieu.c b/stob/ut/adieu.c
index d6f81f1..eaf7068 100644
--- a/stob/ut/adieu.c
+++ b/stob/ut/adieu.c
@@ -165,7 +165,7 @@ static void test_write(int i)
 	struct m0_fol_rec_part *fol_rec_part;
 
 	M0_ALLOC_PTR(fol_rec_part);
-	M0_UT_ASSERT(fol_rec_part != NULL);
+	M0_UB_ASSERT(fol_rec_part != NULL);
 
 	m0_stob_io_init(&io);
 
-- 
1.8.3.2

